torch/fft/__init__.py

```diff
@@ -808,6 +808,271 @@
             -0.5000+0.6882j])
 """.format(**common_args))
 
+hfft2 = _add_docstr(_fft.fft_hfft2, r"""
+hfft2(input, s=None, dim=(-2, -1), norm=None, *, out=None) -> Tensor
+
+Computes the 2-dimensional discrete Fourier transform of a Herimitian symmetric
+:attr:`input` signal. Equivalent to :func:`~torch.fft.hfftn` but only
+transforms the last two dimensions by default.
+
```

Comment:
I think the note from `hfftn` that `input` is interpreted as a one-sided Hermitian signal in the time domain should also be repeated here, otherwise it's unclear what if `input` itself is required to have some symmetric properties.